cmd/plotFunctions/plotFunctions.go

```diff
@@ -92,13 +98,109 @@ func main() {
 		functionArgs := flag.Arg(1)
 		outFile := flag.Arg(2)
 		words := strings.Split(functionArgs, ",")
-		if len(words) != 2 {
+		if len(words) != 3 {
 			log.Fatalf("An allele frequency probability mass function is defined by three parameters, received %d.", len(words))
 		}
 		alpha := common.StringToFloat64(words[0])
 		n := common.StringToInt(words[1])
 		integralError := common.StringToFloat64(words[2])
 		popgen.PlotAfsPmf(alpha, n, outFile, integralError)
+	} else if flag.Arg(0) == "AfsProbabilityAncestral" {
```

Comment:
I know I am a broken record, but this monster of an if else would be much prettier as 
``` 
switch flag.Arg(0) {
     case "AfsProbabilityAncestral" :
              doSomePlotting()
     case "..." :
}
```